articles/obscure-html/index.md

```diff
@@ -0,0 +1,111 @@
+
+```html
+<button onclick="showContent()">Show Me Some Stuff!</button>
+
+<template>
+    <p>Hey look! It's stuff!</p>
+    <img src="stuff.jpg">
+</template>
+
+<script>
+function showContent() {
+    var temp = document.getElementsByTagName("template")[0]; // gets the first template in the page
+    var clon = temp.content.cloneNode(true); // a clone of the template
+    document.body.appendChild(clon); // adds the template to the bottom of the page
+}
+</script>
+```
+
+#### Uses:
+
+* If you need to render the same content many times, but don't want to type it out, you can write some JS code to do it for you
+* If you need to repeat content but change just a little bit, you can change it before inserting it into the page as well
+
+### 2. details
+
+By default, content within the `<details>` tags are hidden, but can be shown by clicking on it. Each element should display a summary of what it's about when it's hidden. Using it looks something like this:
+
+```html
+<details>
+	<summary>Click me to learn about Malaria!</summary>
+	Ok, so here's some more information. Malaria, while being extremely common in some areas, is very easy to prevent. Malaria is mainly spread by mosquitos, which can be kept at bay using mosquito nets. It costs about $2 to buy a net. Donate a couple hundred of those, and you will have easily have saved a life.
+</details>
+```
+
+![A details section before being expanded](details1.png)
+
+![A details section after being expanded](details2.png)
+
+#### Uses:
+
+* This could be used for an FAQ in a fairly obvious way
+* Samsung uses something similar on [their website](https://www.samsung.com/us/shop/all-deals/) for footnotes
+
+### 3. wbr
+
+The `<wbr>` tag can be used to show a possible word break. Sometimes you have a word that's so long that the web browser automatically breaks it up. Using the `<wbr>` tag gives the browser a specific place to break up a word if it's necessary.
+
+```html
+anti<wbr>dise<wbr>stablishmen<wbr>taria<wbr>nism
+```
+
+#### Uses
+* The Hugo theme I use automatically splits up the words in reasonable places for me. Although, I should say that it isn't using the `<wbr>` tag.  My guess is that it's [using CSS](https://css-tricks.com/almanac/properties/w/word-break/) instead. Not all browsers support this feature. You can use it for compatibility with ***checks notes*** [Google Chrome](https://caniuse.com/#feat=css-hyphens).
+* If you have a word that is very very long, like [Lopadotemachoselachogaleokranioleipsanodrimhypotrimmatosilphioparaomelitokatakechymenokichlepikossyphophattoperisteralektryonoptekephalliokigklopeleiolagoiosiraiobaphetraganopterygon](https://en.wikipedia.org/wiki/Lopadotemachoselachogaleokranioleipsanodrimhypotrimmatosilphioparaomelitokatakechymenokichlepikossyphophattoperisteralektryonoptekephalliokigklopeleiolagoiosiraiobaphetraganopterygon) or [this one](https://en.wiktionary.org/wiki/Appendix:List_of_protologisms/Long_words/Titin#Noun), you may want this.
```

Comment:
This actually causes a giant horizontal scroll bar on our site. Could you line break it ?